{{cookiecutter.project_slug}}/server/{{cookiecutter.project_slug}}/core/views.py

```diff
@@ -128,6 +137,40 @@ def reset_password(request, *args, **kwargs):
     return Response(response_data, status=status.HTTP_200_OK)
 
 
+{%- if cookiecutter.include_mobile == 'y' %}
+@api_view(["get"])
```

Comment:
This view does actually create an object, I personally think that POST might be more adequate.
Any specific reason for the GET?

Reply:
nope no reason I just don't change things that already work, since we are using get in the webapp version I used the same here